README.md

```diff
@@ -185,29 +188,34 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
 
    ```shell
    (venv) $ ilab
-   Usage: ilab [OPTIONS] COMMAND [ARGS]...
+   Usage: ilab [GROUP] COMMAND [ARGS]...
 
    CLI for interacting with InstructLab.
 
-   If this is your first time running InstructLab, it's best to start with `ilab init` to create the environment.
+   If this is your first time running InstructLab, it's best to start with `ilab config init` to create the environment.
 
    Options:
    --config PATH  Path to a configuration file.  [default: config.yaml]
    --version      Show the version and exit.
    --help         Show this message and exit.
 
-   Commands:
-   chat      Run a chat using the modified model
-   check     (Deprecated) Check that taxonomy is valid
-   convert   Converts model to GGUF
-   diff      Lists taxonomy files that have changed since <taxonomy-base>...
-   download  Download the model(s) to train
-   generate  Generates synthetic data to enhance your example data
-   init      Initializes environment for InstructLab
-   list      (Deprecated) Lists taxonomy files that have changed since <taxonomy-base>.
-   serve     Start a local server
-   test      Runs basic test to ensure model correctness
-   train     Takes synthetic data generated locally with `ilab generate`...
+   Command: 
+      config      Command group for Interacting with the Config of InstructLab
+      data        Command group for Interacting with the Data of generated by...
+      model       Command group for Interacting with the Models in InstructLab
+      sysinfo     Print system information
+      taxonomy    Command group for Interacting with the Taxonomy in InstructLab
+
+      Aliases:
+      chat: model chat
+      convert: model convert
+      diff: taxonomy diff
+      download: model download
+      generated: model generate
```

Comment:
Since you are going through the effort of changing those, please consider a this alternative as well (less repetitive and less generic): 

```suggestion
 Commands:
  config      Configure InstructLab settings
  data        Manage generated data
  model       Operations on models (chat, convert, download, serve, test, train)
  sysinfo     Display system information
  taxonomy    Handle taxonomy data

Aliases:
  chat        Shortcut for 'model chat'
  convert     Shortcut for 'model convert'
  diff        Shortcut for 'taxonomy diff'
  download    Shortcut for 'model download'
  generate    Shortcut for 'model generate'
  init        Shortcut for 'config init'
  serve       Shortcut for 'model serve'
  test        Shortcut for 'model test'
  train       Shortcut for 'model train'
```